src/Conference/Admin/Menu.php

```diff
@@ -99,5 +132,14 @@ public function organize_post_types() {
 			'read',
 			'post-new.php?post_type=' . Plugin::SPONSOR_POSTTYPE
 		);
+		// Submenu for Sponsor Levels Taxonomy.
+		add_submenu_page(
+			$this->menu_slug,
```

Comment:
Would love to use `$this->get_menu_slug()` it allows for more flexibility later.